src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
"Undefined behaviour" sounds quite disturbing, especially for this kind of stuff that, when missed, can just go completely wrong.

Is there a case when users use NODEFS and TTY directly without depending on FS? If not, then it shouldn't be a problem anyway, but if they do, maybe better to split utils used by other pieces into a separate variable and explicitly depend on it? This would solve a problem and reduce generated code for such cases for free.